udagram-api-user/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.8'
+services:
+  postgres:
+    image: postgres:latest
```

Comment:
Again, the version should be better. Moreover, if you have already a docker-compose file in the root, there's no need to have duplicate individuals. You can just spin one service using `docker-compose up SERVICE_NAME -d`. Therefore, you can remove this file